Dropbox docs

ivov opened the pull request:
Here are the Dropbox docs. The new example workflow is [here](https://n8n.io/workflows/439).

I also added more details to the Dropbox credentials section.

Let me know if you have any comments.

smamudhan commented on docs/nodes/credentials/Dropbox/README.md:
```diff
@@ -1,16 +1,19 @@
-3. Click on "Create App".
-4. Use provided credentials with your Dropbox node credentials in n8n.
+1. Access the [Dropbox app creation page](https://www.dropbox.com/developers/apps/create).
+2. Select Dropbox API and scoped access.
```
Could you divide this into two statements?
Eg: Under the 'Choose an API' section, select *Dropbox API*

smamudhan replied:
"scoped access" needs to be capitalized to be "Scoped access"

smamudhan commented on docs/nodes/credentials/Dropbox/README.md:
```diff
@@ -1,16 +1,19 @@
-4. Use provided credentials with your Dropbox node credentials in n8n.
+1. Access the [Dropbox app creation page](https://www.dropbox.com/developers/apps/create).
+2. Select Dropbox API and scoped access.
+3. Select access type, enter the app name and accept the terms and conditions.
```
Could you please change it to match the above comments?

smamudhan commented on docs/nodes/credentials/Dropbox/README.md:
```diff
@@ -1,16 +1,19 @@
+1. Access the [Dropbox app creation page](https://www.dropbox.com/developers/apps/create).
+2. Select Dropbox API and scoped access.
+3. Select access type, enter the app name and accept the terms and conditions.
+4. Click on Create App.
```
"Create App" needs to be updated to be "Create app"

smamudhan commented on docs/nodes/credentials/Dropbox/README.md:
```diff
@@ -1,16 +1,19 @@
+2. Select Dropbox API and scoped access.
+3. Select access type, enter the app name and accept the terms and conditions.
+4. Click on Create App.
+5. In the Dropbox app dashboard, navigate to Settings, scroll down to OAuth2 and click Generate under Access token.
```
Could you change it to: click on the 'Generate' button under 'Generated access token'

smamudhan commented on docs/nodes/credentials/Dropbox/README.md:
```diff
@@ -1,16 +1,19 @@
+3. Select access type, enter the app name and accept the terms and conditions.
+4. Click on Create App.
+5. In the Dropbox app dashboard, navigate to Settings, scroll down to OAuth2 and click Generate under Access token.
+6. In the Dropbox app dashboard, navigate to Permissions, and select the permissions your workflow will need.
```
Could you add "and click on the Submit button" ?

smamudhan commented on docs/nodes/credentials/Dropbox/README.md:
```diff
@@ -1,16 +1,19 @@
+4. Click on Create App.
+5. In the Dropbox app dashboard, navigate to Settings, scroll down to OAuth2 and click Generate under Access token.
+6. In the Dropbox app dashboard, navigate to Permissions, and select the permissions your workflow will need.
+7. Return to n8n and enter the access token in your n8n Dropbox credentials.
```
Could you update this to:  
Return to n8n and enter the Access Token in your Dropbox API credentials.

smamudhan commented on docs/nodes/nodes-library/nodes/Dropbox/README.md:
```diff
@@ -0,0 +1,48 @@
+	- Move a file
+	- Upload a file
+- Folder
+	- Copy a file
+	- Create a file
+	- Delete a file
+	- List files and folders in the folder
+	- Move a folder
```
Could you update it to match:
![image](https://user-images.githubusercontent.com/48641776/86330794-d1744c00-bc65-11ea-8550-c46808183291.png)

smamudhan commented on docs/nodes/nodes-library/nodes/Dropbox/README.md:
```diff
@@ -0,0 +1,48 @@
+
+### 2. Dropbox node
+
+1. Click on the plus button, search for "Dropbox", select it and double-click on the new node.
+2. Select "Create new" in the *Dropbox API* field.
+3. Create a credential as shown in [this tutorial](../../../credentials/Dropbox/).
+4. Enter the name of the new folder *Folder* field.
+5. Click on *Execute Node* to run the workflow.
```
Could you add an empty newline to the end of the file?

smamudhan replied:
Could you remove steps 1 & 2; and change this to look similar to https://docs.n8n.io/nodes/n8n-nodes-base.pagerDuty/#example-usage

smamudhan commented on docs/nodes/nodes-library/nodes/Dropbox/README.md:
```diff
@@ -0,0 +1,48 @@
+1. Click on the plus button, search for "Dropbox", select it and double-click on the new node.
+2. Select "Create new" in the *Dropbox API* field.
+3. Create a credential as shown in [this tutorial](../../../credentials/Dropbox/).
+4. Enter the name of the new folder *Folder* field.
```
Could you fix the grammatical error?

ivov commented:
Hey @smamudhan, thanks for checking it!

> Eg: Under the 'Choose an API' section, select *Dropbox API*

To look more like the others, I think UI labels in the various services seem to use no quotes and no italics, only caps according to the original, so I changed the example like that. Let me know otherwise - I'm not really sure what the rule is.

> Could you add an empty newline to the end of the file?

I have an empty newline locally but I see now up at GitHub it's disappeared. I just added two new empty newlines to see if that fixes it. Also you didn't mention the credentials file so I didn't change it, but for consistency maybe it should be changed as well?

tanay1337 commented:
Hey @ivov, thanks for making the changes. In case the credentials file does not have a empty newline at the end, please feel free to add it. You are correct about consistency there. In terms of styling, @smamudhan's suggestion makes sense. But yeah, we do need to make the rest of the credentials consistent to this style eventually.

ivov commented:
Thanks! I'll make the styling and newline changes and push again, here and in the PayPal PR.

ivov commented:
I added two final newlines locally because when I added one, the end result at GitHub had none. Adding two seems to have fixed it as I now see only one in the PR - otherwise let me know and I'll look into it.

tanay1337 commented:
Merged, thanks a lot @ivov 🎉
